api/src/main/java/org/openmrs/validator/ConditionValidator.java

```diff
@@ -44,5 +45,12 @@ public void validate(Object object, Errors errors) {
 		if(!(object instanceof Condition)){
 			throw  new IllegalArgumentException("The object parameter should be of type " + Condition.class);
 		}
+		Condition condition = (Condition) object;
+		if(condition.getCondition() == null){
+			throw new APIException("The condition parameter is required");
```

Comment:
It's better to reject the field value rather than to throw the exception for proper handling of external code

Reply:
Done